pmlc/conversion/tile_to_pxa/specials.cc

```diff
@@ -675,6 +702,39 @@ struct GatherOpConversion : public OpConversionPattern<tile::GatherOp> {
     auto valuePlusHalf = rewriter.create<mlir::AddFOp>(loc, value, half);
     return floorFPToSI(loc, rewriter, valuePlusHalf, integerType);
   }
+
+  Value processPaddingMode(Location loc, ConversionPatternRewriter &rewriter,
+                           Value result, Value idx, IndexBounds bounds,
+                           Type elementType, PaddingMode paddingMode) const {
```

Comment:
The input `Value result` for this function is already the interpolated value treat the data as if it was edge padded. So if the `paddingMode` is edge, we can just return this value.